### PR TITLE
Adding flaremodel

### PR DESCRIPTION
**Pull request by ydallilar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by anilk666** [comment]
> Checklist
> 
> * [x]  Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
> * [x]  License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
> * [x]  Source is from official source.
> * [x]  Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
> * [x]  If static libraries are linked in, the license of the static library is packaged.
> * [x]  Build number is 0.
> * [x]  A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
> * [x]  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
> * [x]  When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

https://github.com/conda-forge/staged-recipes/pull/17147#issue-1070097635
anilk666

**Comment by ydallilar** [comment]
I am willingly listed as sole maintainer of this package.

**Comment by ydallilar** [comment]
@conda-forge/help-python, ready for review.

**Comment by ydallilar** [comment]
@conda-forge/help-python ready to review.

**Comment by carterbox** [comment]
conda-forge-admin, please restart ci

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/flaremodel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  skip: true  # [win and vc<14]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - libblas
+    - libcblas
```
[the remark this answers is not in the document]
Comment (ydallilar) [reply recipes/flaremodel/meta.yaml:24]: It has been a while. I dont remember. I will check in a few days.

[the remark this answers is not in the document]
Comment (ydallilar) [reply recipes/flaremodel/meta.yaml]: The C library I am including actually doesn't require blas at all. But, the build fails without. I am assuming it has to do with cython, numpy (or possibly conda-forge blas implementation that comes with it: https://conda-forge.org/docs/maintainer/knowledge_base.html#blas).

**Pull request closed without merging**